pkg/controller/runtime-util.go

```diff
@@ -0,0 +1,121 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+
+	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/operator"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+// MakeEmptyCDIConfigSpec creates cdi config manifest
+func MakeEmptyCDIConfigSpec(name string) *cdiv1.CDIConfig {
+	config := &cdiv1.CDIConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+			Labels: map[string]string{
+				common.CDILabelKey:       common.CDILabelValue,
+				common.CDIComponentLabel: "",
+			},
+		},
+	}
+	return config
+
+}
+
+// SetOwnerToOperator sets the object owner to the same owner as the cdi config map.
+func SetOwnerToOperator(client *client.Client, object metav1.Object, namespace string) error {
```

Comment:
`client.Client` is an interface so client param prob shouldn't be a pointer.  But I propose getting rid of this function entirely and using `SetOwner` in pkg/operator.  The config controller can have a client-go client in addition to controller runtime client.

Reply:
let me check.

Reply:
Switched the create to use the client-go clients and made an init just because I could.